content/python/concepts/built-in-functions/terms/locals/locals.md

```diff
@@ -0,0 +1,50 @@
+---
+
+Title: 'locals()'
+Description: 'The `locals()` function returns a dictionary with all the data necessary to run the program.'
+Subjects: 
+- 'Devloper tools'
+- 'Computer science'
+Tags:
+- 'Dictionary' 
+- 'Metadata' 
+- 'Symbol'
+Catalog-content: 
+- 'learn python 3'
+- 'paths/computer-science'
+
+---
+
+The `locals()`function returns a dictionary with all the data necessary to run the program.
+
+## Syntax
+
+```py
+locals()
+```
+
+## Example
+
+```py
+{'In': ['', 'locals()'],
+ 'Out': {},
+ '_': '',
+ '__': '',
+ '___': '',
+ '__builtin__': ,
+ '__builtins__': ,
+ '__name__': '__main__',
+ '_dh': ['/home/repl'],
+ '_i': '',
+ '_i1': 'locals()',
+ '_ih': ['', 'locals()'],
+ '_ii': '',
+ '_iii': '',
+ '_oh': {},
+ '_sh': ,
+ 'exit': ,
+ 'get_ipython': >,
+ 'quit': }
```

Comment:
> That's just an example. I wasn't sure what the actual result would be so I looked up the result elsewhere.
> 
> ...Was I supposed to write out a function? I thought I was just supposed to show the result of the code.

Basically example section should have a code that gives an output, so this could be replaced with a code having `locals` that gives an output